ext/arm/arm_math_types.h.in

```diff
@@ -11,18 +11,11 @@
 #ifndef _ARM_MATH_TYPES_WRAPPER_H_
 #define _ARM_MATH_TYPES_WRAPPER_H_
 
-#define ARM_MATH_CM{{core}}
-#define __ARM_FEATURE_MVE 0
-
-%% if with_fpu
-#ifndef __FPU_PRESENT
-#define __FPU_PRESENT 1
-#endif
-%% endif
+#include "modm/platform/device.hpp"
```

Comment:
```suggestion
#include <modm/platform/device.hpp>
```

Reply:
Out of curiosity, why the angle brackets in this case?

Reply:
So that we can move the arm_math_types.h.in file around and not worry about this include. (The <> looks in the include paths, the "" just locally.) Just less fragile and also <> has been the convention in all other files. 

I'm not even sure why this is even working, cos this file is generated to `modm/ext/cmsis/dsp`, and `modm/ext/cmsis/dsp/modm/platform/device.hpp` doesn't even exist?

Reply:
Huh, seems like there are some weird rules for includes

https://stackoverflow.com/a/21594/2996689
https://gcc.gnu.org/onlinedocs/cpp/Search-Path.html